README.md

```diff
@@ -7,26 +7,11 @@ Ruby on Rails app that handles the submission of lettings and sales of social ho
 
 ## Domain documentation
```

Comment:
I'd maybe just add a little bit to the main readme about how to spin up the documentation website locally